### ISSUE TITLE
ValueError: '.ipynb' is not a valid Extension

### ISSUE THREAD

**Issue by tiginamaria** [hyperskill/hyperstyle#156]
I have tried to use Hyperstyle as a linter in a real python project. I ran Hyperstyle on a directory with `.ipynb` files besides `.py`, but failed with exceptions`ValueError: '.ipynb' is not a valid Extension`. Same was with  `.md` files in diectoy: `ValueError: '.md' is not a valid Extension`. The full traceback was following:
```
Traceback (most recent call last):
  File "/Users/tiginamaria1999/PycharmProjects/hyperstyle/hyperstyle/src/python/review/run_tool.py", line 169, in main
    n_issues = perform_and_print_review(args.path, OutputFormat(args.format), config)
  File "/Users/tiginamaria1999/PycharmProjects/hyperstyle/hyperstyle/src/python/review/reviewers/perform_review.py", line 57, in perform_and_print_review
    review_result = perform_review(path, config)
  File "/Users/tiginamaria1999/PycharmProjects/hyperstyle/hyperstyle/src/python/review/reviewers/perform_review.py", line 85, in perform_review
    metadata = explore_project(path)
  File "/Users/tiginamaria1999/PycharmProjects/hyperstyle/hyperstyle/src/python/review/reviewers/utils/metadata_exploration.py", line 75, in explore_project
    inner_files.append(explore_file(file_path))
  File "/Users/tiginamaria1999/PycharmProjects/hyperstyle/hyperstyle/src/python/review/reviewers/utils/metadata_exploration.py", line 65, in explore_file
    language = guess_file_language(path)
  File "/Users/tiginamaria1999/PycharmProjects/hyperstyle/hyperstyle/src/python/review/common/language.py", line 54, in guess_file_language
    return EXTENSION_TO_LANGUAGE.get(Extension.get_extension_from_file(file_path), Language.UNKNOWN)
  File "/Users/tiginamaria1999/PycharmProjects/hyperstyle/hyperstyle/src/python/review/common/file_system.py", line 40, in get_extension_from_file
    return Extension(os.path.splitext(file)[1])
  File "/usr/local/Cellar/python@3.9/3.9.12/Frameworks/Python.framework/Versions/3.9/lib/python3.9/enum.py", line 384, in __call__
    return cls.__new__(cls, value)
  File "/usr/local/Cellar/python@3.9/3.9.12/Frameworks/Python.framework/Versions/3.9/lib/python3.9/enum.py", line 702, in __new__
    raise ve_exc
ValueError: '.ipynb' is not a valid Extension
```

 It seems that Hyperstyle fails when meets file with extension out of enum class `hyperstyle.src.python.review.common.file_system.Extension`. But I expected that Hyperstyle will skip such files and give feedback only on `.py` files it meets. 

Maybe you can do something with it?

**Comment by GirZ0n** [comment]
I think we can add Extension.UNKNOWN here:
https://github.com/hyperskill/hyperstyle/blob/e10a38c40e1195b12e0af02285163755265731bd/hyperstyle/src/python/review/common/file_system.py#L28-L40

And here a new pair of EXTENSION.UNKNOWN: LANGUAGE.UNKNOWN
https://github.com/hyperskill/hyperstyle/blob/e10a38c40e1195b12e0af02285163755265731bd/hyperstyle/src/python/review/common/language.py#L44-L50

And then Hyperstyle should not crash in that case. @nbirillo, what do you think?

**Comment by nbirillo** [comment]
I think return None is better

**Comment by GirZ0n** [comment]
Fixed in #157
